mrp_repair_order/models/repair.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Antoni Marroig(APSL-Nagarro)<amarroig@apsl.net>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class RepairOrder(models.Model):
+    _inherit = "repair.order"
+
+    mrp_count = fields.Integer(
+        "Mrp production order count", compute="_compute_mrp_production_count"
+    )
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        self.env["mrp.production"].browse(
+            self.env.context["default_mrp_id"]
+        ).repair_id = res.id
```

Comment:
This will fail if more then one repair order is created, res will contain multiple records